### DISCUSSION TITLE
Model the inside of the crew hideaway place in cabin

### DISCUSSION THREAD

**Discussion by Simleybob** [magknight/787aviatorsTracker#759] [Ideas]
Could be cool, in here so you can see the basic structure of beds etc 
![image](https://user-images.githubusercontent.com/81443205/157736085-695d066d-f26d-4123-b20a-d372e814ab1e.png)

**Comment by CaptainJankez** [comment]
Yup, here are some real photos from the upper crew restroom 
![image](https://user-images.githubusercontent.com/91334772/158030569-29c52794-48eb-4add-930a-e502d56dde51.png)
![image](https://user-images.githubusercontent.com/91334772/158030579-ebf30666-436e-44fd-b303-69462ee2dd74.png)
![image](https://user-images.githubusercontent.com/91334772/158030587-a6f897a4-3e03-4fc2-a262-811431a89474.png)
![image](https://user-images.githubusercontent.com/91334772/158030595-059a7cba-5e4e-484a-b9d1-a41eacbc064c.png)
![image](https://user-images.githubusercontent.com/91334772/158030613-95fc4671-c599-4a88-a0e7-06ff06400fe9.png)
Those photos aren't mine ofc

**Reply by Simleybob** [reply]
Thanks 😊
